ReactSkia/components/RSkComponentText.cpp

```diff
@@ -42,6 +42,27 @@ inline SkScalar yPosOffset(AttributedString attributedString, SkScalar paraHeigh
     return 0;
 }
 
+inline void ellipSize(std::shared_ptr<ParagraphBuilder> paraBuilder, std::vector<LineMetrics>& metrics, ParagraphAttributes paragraphAttributes, Rect frame) {
```

Comment:
if only "paragraphAttributes.maximumNumberOfLines" is been used in this function, can we pass paragraphAttributes.maximumNumberOfLines as argument instead of paragraphAttributes
same applies to "frame"

Reply:
Updated.